sample-apps/react/react-video-demo/src/App.tsx

```diff
@@ -59,6 +67,16 @@ const Init = () => {
 
   const { edges, fastestEdge } = useEdges(client);
 
+  useEffect(() => {
+    requestMediaPermissions()
```

Comment:
I propose to keep loading state meanwhile we do not know, whether the browser grants permissions. That loading state can be then represented on the video preview in the lobby. Currently, when I load the page, first I see, that I do not have permissions, even though they are already granted and then the video or video fallback is shown.

Reply:
We should probably check for microphone and video permissions separately.